src/http_checker.cpp

```diff
@@ -1,5 +1,6 @@
 
 #include "http_checker.h"
+#include <cstring>
```

Comment:
Non blocking: Ideally this should be moved to `http_checker.h` header file along with the rest includes.

Reply:
I was going for "minimal changes to get it to compile". 😆 thanks.